app/Http/Controllers/AuthController.php

```diff
@@ -46,22 +49,91 @@ public function login(Request $request)
      */
     public function register(Request $request)
     {
-        $validated = $request->validate([
-            'name' => 'required|max:255',
-            'email' => 'required|email|max:255|unique:users,email',
-            'password' => 'required|confirmed|min:6',
-        ]);
+      
+     
+            $validated = $request->validate([
+                'name' => 'required|max:255',
+                'email' => 'required|email|max:255|unique:users,email',
+                'password' => 'required|confirmed|min:6',
+                'image_url' => 'nullable|image|mimes:jpeg,jpg,png,svg|max:2048'
+            ]);
+         
 
-        $validated['password'] = Hash::make($validated['password']);
+           
+            $validated['password'] = Hash::make($validated['password']);
 
-        $user = User::create($validated);
+            $dataWithImage = new UploadImage();
+            $userImage = new User();
+            $finalDataUser = $dataWithImage->storeAndUpdateImageUser($validated,$userImage,$folderNameUserImage = "user");
+        //   dd($finalDataUser);
+            $user = User::create($finalDataUser);
 
-        return response()->json([
-            'access_token' => $user->createToken('api_token')->plainTextToken,
-            'token_type' => 'Bearer',
-        ], 201);
+            return response()->json([
+                'status' => 1,
+                'status_message' => 'Successfully registered user',
+                'access_token' => $user->createToken('api_token')->plainTextToken,
+                'token_type' => 'Bearer',
```

Comment:
remove unnecessary information in JSON after successful login: status, status_message and user details.

Reply:
ok did it